plugins/postcss-custom-media/src/transform-at-media/transform-at-media.ts

```diff
@@ -1,5 +1,5 @@
 import { alwaysTrue, neverTrue } from './always-true-or-false';
-import { isGeneralEnclosed, isMediaAnd, isMediaConditionList, isMediaFeature, isMediaFeatureBoolean, isMediaNot, isMediaOr, isMediaQueryInvalid, isMediaQueryWithType, MediaQuery, newMediaFeaturePlain, parse } from '@csstools/media-query-list-parser';
+import { isGeneralEnclosed, isMediaAnd, isMediaConditionList, isMediaFeature, isMediaFeatureBoolean, isMediaNot, isMediaOr, isMediaQueryInvalid, isMediaQueryWithType, MediaFeature, MediaQuery, newMediaFeaturePlain, parse } from '@csstools/media-query-list-parser';
```

Comment:
[nit] Should we make this a line per import so it's easier to see/maintain?

Reply:
Good catch!

The auto import feature in typescript makes me lazy.
It just keeps adding things to the list and I never really look at it :D